libraries/app/database_api.cpp

```diff
@@ -928,6 +924,8 @@ std::map<std::string, full_account> database_api_impl::get_full_accounts( const
       uint64_t api_limit_get_full_accounts_lists = _app_options->api_limit_get_full_accounts_lists;
 
       // Add the account's proposals
+      const auto& proposal_idx = _db.get_index_type< primary_index< proposal_index > >();
```

Comment:
Why did you move this inside the for loop?

Reply:
My mistake, i didn't saw the implication of the move correctly, my apologies. reverted at https://github.com/bitshares/bitshares-core/pull/1749/commits/ed53cb09fd677fa95d01cd3ad10b0d2fe0acd924